Flatr-App-Core.cabal

```diff
@@ -36,7 +36,8 @@ executable Flatr-App-Core-exe
                      , word8
                      , bytestring
                      , base16-bytestring >=0.1
-                     , cryptohash >=0.11
+                     , cryptonite >=0.24
+                     , memory
```

Comment:
What is memory required for?

Reply:
For Data.ByteArray.convert, which is used to convert the Context given by hashFinalize to a ByteString.

Reply:
No longer needed, deleted from the cabal file.